### ISSUE TITLE
Scope Mind indicator when set.

### ISSUE THREAD

**Issue by Iznik285** [dvorka/mindforger#1289]
Would it be possible to indicate on the menu bar when the Scope Mind filter is set?

I am using Windows 10 / MF 1.52.0 and UI theme - light (but it should work on all themes)

**Comment by dvorka** [comment]
@Iznik285 Nice idea. Do you mean that (for example) icon color would be different in toolbar in case that mind filter would be active?

**Comment by Iznik285** [comment]
Yes, exactly. It is just to make it clearer that the filter is enabled when looking at the toolbar. 
It is still useful to have the indicate at the bottom as well.
